pkgs/desktops/plasma-5/kwin/default.nix

```diff
@@ -37,6 +37,7 @@ mkDerivation {
   patches = [
     ./0001-follow-symlinks.patch
     ./0002-xwayland.patch
+    ./0001-NixOS-Unwrap-executable-name-for-.desktop-search.patch
```

Comment:
```suggestion
    ./0003-NixOS-Unwrap-executable-name-for-.desktop-search.patch
```

Reply:
Now that's grasping at straws. `git format-patch` handles naming the file from the commit message. The prefix is to order patches *in a group*, so that e.g. patch 0002 happens after 0001, because the reverse is not necessarily true.

`0001` here is fine as it is not linked to the other two patches. It can be applied before or after the other two patches.

Or, let's say, let's see what the maintainer of the package thinks about that. But forcing arbitrary naming of the patch files compared to what the tooling gives is going to make it more awkward to work woth.

Reply:
Shouldn't git name them continues when you export them all at once?

Reply:
I think @samueldr is not applying all the patches at once. I usually do `git am *.patch` when I'm working on a package. I don't have feelings about the file name. I expect it will get renamed the next time anyone works on this package's patches, and that's fine.